### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by Rash-598**
Added release notes for 2.6.2-preview module

### CODE REVIEW

**Review by shriya999 — CHANGES_REQUESTED** [review]
could you remove the 2,3,4 points since they are same as in 2.6.1-preview?

Edit : and also remove the add-teamChannelUser point since they were moving it to GA - so it is not applicable to a preview release?

**Comment by Rash-598** [comment]
> could you remove the 2,3,4 points since they are same as in 2.6.1-preview?
> 
> Edit : and also remove the add-teamChannelUser point since they were moving it to GA - so it is not applicable to a preview release?

Done

**Review by shriya999 — APPROVED** [review]

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
 
 | Date | Version | Updates |
 |------- | -------------------- | ------------------------------ |
+| November 2021 | [2.6.2-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.6.2-preview) |<li>Releases modernized versions of all 75+ Grant-Cs<Name>Policy cmdlets (all parameter sets). These are expected to work similar to their remoting counter parts. The modern versions will be gradually rolled out, tenants that aren't flighted will see the older remoting version.</li><li>Releases modernized versions of all 75+ Remove-Cs<Name>Policy cmdlets. These are expected to work similar to their remoting counter parts. The modern versions will be gradually rolled out, tenants that aren't flighted will see the older remoting version.</li><li>Releases Set-CsUserCallingSettings cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Export-CsOnlineAudioFile cmdlet.</li><li>Fixes error handlings for [Get\|Import\|Remove]-CsOnlineAudioFile cmdlets.</li>
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| November 2021 | [2.6.2-preview](https://www.powershellgallery.com/packages/MicrosoftTeams/2.6.2-preview) |<li>Releases modernized versions of all 75+ Grant-Cs<Name>Policy cmdlets (all parameter sets). These are expected to work similarly to their remoting counterparts. The modern versions will be gradually rolled out, tenants that aren't flighted will see the older remoting version.</li><li>Releases modernized versions of all 75+ Remove-Cs<Name>Policy cmdlets. These are expected to work similarly to their remoting counterparts. The modern versions will be gradually rolled out, tenants that aren't flighted will see the older remoting version.</li><li>Releases Set-CsUserCallingSettings cmdlet. This cmdlet will be eventually released in GA module. Releasing the trial version in preview module.</li><li>Releases Export-CsOnlineAudioFile cmdlet.</li><li>Fixes error handlings for [Get\|Import\|Remove]-CsOnlineAudioFile cmdlets.</li>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Rash-598, please commit the suggested changes. Many thanks.

**Comment by Rash-598** [comment]
Done @JohanFreelancer9 please check

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Rash-598** [comment]
@tiburd can you please review and merge this PR

**Pull request merged**